### ISSUE TITLE
How to Enable and Disable PushNotifications for particular User?

### ISSUE THREAD

**Issue by ajay2c9** [QuickBlox/quickblox-ios-sdk#176]
I had a functionality that i can block any user in my following list, i am showing the following users list for same chat users list in my app.
Now I had two chat rooms, one is 1-1 chat room and another is Group chat room.

1. I Blocked the user in 1-1 chat room, now i don’t need to receive the notifications from that particular user. Until i Unblock the user. Is this Functionality possible from Quickblox SDK side? how?

2. i am doing the same for Group Chat also. Is it possible to do this ?

**Comment by AndreyMoskvin** [comment]
Yes, it is possible you need to use PrivacyLists and block user by identifier and for group chat.

**Comment by ZalakSA** [comment]
@Berk0ld Hello,

Can you please help me how I can prevent sending notification when user is blocked. I am able to stop local notification but not push notification.

Thanks

**Comment by AndriiMoskvin** [comment]
@ZalakSA  - sorry, I can't help you I no longer work in Quickblox.

**Comment by ZalakSA** [comment]
I am badly stuck with this. If you have gone anything such then or anybody else can help me for this would be greatful.

Thanks

**Comment by Raikerian** [comment]
Hi @ZalakSA 

Please create a separate issue and describe your problem with more details.
